index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <title>International Mexican Food Convention</title>
+</head>
+<body>
+    <header>
+        <div id="blackHeader">
+            <div class="blackIconsContainer">
+                <a href=""><img class="blackHeaderIcon" src="./images/facebookIcon.png" alt=""></a>
+                <a href=""><img class="blackHeaderIcon" src="./images/twitterIcon.png" alt=""></a>
+            </div>
+            <a href=""><div class="blackHeaderOption">English</div></a>
+            <a href=""><div class="blackHeaderOption">My Page</div></a>
+            <a href=""><div class="blackHeaderOption">Logout</div></a>
+        </div>
+        <div id="whiteHeader">            
+            <a href="./index.html"><img id="logoHeader" src="./images/logoOrangeMx-removebg-preview.png" alt="logo"></a>
+            <div id="menuOptions">
+                <a class="menuOptionsLinks" href="./about.html"><div>About</div></a>
+                <a class="menuOptionsLinks" href="./index.html"><div>Event</div></a>
+                <a class="menuOptionsLinks" href="#programDetails"><div>Join</div></a>
+                <a class="menuOptionsLinks sponsorMobile" href="./about.html#footerOne"><div>Sponsor</div></a>
+                <a class="menuOptionsLinks sponsorDesk" href="#footerOne"><div>Sponsor</div></a>
+                <a class="menuOptionsLinks" href="#"><div>News</div></a>
+                <a class="menuOptionsLinks" href="#"><div>CC Campaign</div></a>
+            </div>
+        </div>
+    </header>
```

Comment:
- [ ] Your navbar compared to the design template is different in that the `CC Campaign` button has a thick red color border. 
- [ ] Also, your navbar has two sections, the black top and the bottom white. The bottom white seems to be short in terms of height. Kindly increase a bit to match the given template.

The template
<img width="623" alt="image" src="https://github.com/AdrianScharff/1st-Capstone-project-MexicanFood/assets/84156714/96e97ee0-68ca-4324-bfe9-533db3bdd604">
 
Your design
<img width="911" alt="image" src="https://github.com/AdrianScharff/1st-Capstone-project-MexicanFood/assets/84156714/34494f33-100d-4db5-be3d-b25f441c0f87">

- [ ] I noticed that your average font-size is around `0.5-0.6rem` which is about `8px` which is too small and not readable for many users who have eye issues. For improved accessibility and readability, I suggest that you increase font-size to about `12px, 14px, 16px` as the case may be.

Reply:
You didn't specify if I have to change the font-size just on the desktop version or also in mobile, so I changed the font-size in both versions, the text is now higher than each of the images, but I can't also icrease the size of the image as there is no more space
![higherthn](https://github.com/AdrianScharff/1st-Capstone-project-MexicanFood/assets/134537383/5da79251-afbe-4364-8b28-5b53ff5401e6)

Reply:
I do not think this matters. You could leave as is.